### PR TITLE
enh(UI): Remove tooltips and add time tick above the graph

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This removes the metrics values tooltip and displays the date time tick over the graph
https://www.loom.com/share/cf98e4f42f964080a0b29634a57130c3

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x (master)

<h2> How this pull request can be tested ? </h2>

- Select a Resource with data
- Select the Graph tab on the panel
- Click on the "Graph Options" icon
- -> The "Display metrics values tooltips" does appear
- Hover the graph part with data
- -> A date time text appears above the graph (which the mouse position on the X axis) 

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Comment by Thebarda** [comment]
The performances are greatly improved with a lot of services graphs for a hosts
In the video below, my host have 100 services with performance data
https://www.loom.com/share/af13acca62ca486f83d19d66b0461aa9

File: www/front_src/src/Resources/Graph/Performance/ExportableGraphWithTimeline/useMousePosition.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+type MousePosition = [number, number] | null;
+
+interface UseMousePosition {
```
Comment (bdauria) [inline www/front_src/src/Resources/Graph/Performance/ExportableGraphWithTimeline/useMousePosition.tsx:5]:
```suggestion
interface MousePositionState{
```

This maybe isn't the best name, but that's the convention we used this far

File: www/front_src/src/Resources/Graph/Performance/ExportableGraphWithTimeline/useMousePosition.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  mousePosition: MousePosition;
+  setMousePosition: React.Dispatch<React.SetStateAction<MousePosition>>;
+}
+
+const useMousePosition = (): UseMousePosition => {
```
Comment (bdauria) [inline www/front_src/src/Resources/Graph/Performance/ExportableGraphWithTimeline/useMousePosition.tsx:10]:
```suggestion
const useMousePosition = (): MousePositionState=> {
```

File: www/front_src/src/Resources/Graph/Performance/useGraphIntersection.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+interface UseIntersection {
```
Comment (bdauria) [inline www/front_src/src/Resources/Graph/Performance/useGraphIntersection.ts:3]:
```suggestion
interface InteractionState{
```

File: www/front_src/src/Resources/Graph/Performance/useGraphIntersection.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  isInViewport: boolean;
+  setElement: React.Dispatch<React.SetStateAction<HTMLElement | null>>;
+}
+
+export const useIntersection = (): UseIntersection => {
```
Comment (bdauria) [inline www/front_src/src/Resources/Graph/Performance/useGraphIntersection.ts:8]:
```suggestion
export const useIntersection = (): InteractionState=> {
```

**Comment by bdauria** [comment]
> The performances are greatly improved with a lot of services graphs for a hosts
> In the video below, my host have 100 services with performance data
> https://www.loom.com/share/af13acca62ca486f83d19d66b0461aa9
> 
> UPDATE: I've tried with a host with > 300 services and it seems that there is no more "increasing lag"

Excellent job on this!

**Review by bdauria — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**